### ISSUE TITLE
LTI: student names are not human-readable in CSV exports

### ISSUE THREAD

**Issue by echuber2** [PrairieLearn/PrairieLearn#2480]
If I export any stats CSVs for an LTI assignment, the student names exported are never the human-readable ones, although that information can be seen in the web interface.

**Comment by trombonekenny** [comment]
The `uid` for LTI created users is the anonymous unique identifier the source platform supplies. It doesn't traditionally look like a username, where other places we use email-like addresses for the uid. Perhaps you're thinking of the `name` variable that is shown in the online views, but not necessarily in the CSVs?

**Comment by echuber2** [comment]
Yes, that's what I mean. Supposing I export a CSV from an LTI item as a backup of the grades, I'd like some way to know from looking at it who is who. Right now, there's no way to tell. Would the fix be to attach at least the name column for this particular case?

**Comment by echuber2** [comment]
I'll just note that the `instances` export does include a real name column, so it works for now. It just might make sense to include that for LTI items where it otherwise wouldn't be.

**Comment by nwalters512** [comment]
@echuber2 is this still a problem for your courses using LTI 1.1?

**Comment by echuber2** [comment]
Yes, I think this is still an issue, although we may want to try to use 1.3 on Coursera soon. For example, if I get "scores by username" here: https://us.prairielearn.com/pl/course_instance/154163/instructor/assessment/2421321/downloads

Then the export looks something like this:

```csv
Username,Homework 1
6a5c9ae77caf4e8496cc,1
```

Similarly, if you get the log at the bottom here:
https://us.prairielearn.com/pl/course_instance/154163/instructor/assessment_instance/7739312
Then there are no student names, and the "Auth user" column has the random LTI identity string.

If this is resolved in LTI 1.3 then it may be worth it for us to just migrate if possible.

**Comment by nwalters512** [comment]
We would very strongly encourage you to migrate to LTI 1.3 at your earliest convenience. It solves lots of issues (including this one) with both the student and instructor experience.
